### PR TITLE
Add black and isort to eth2 code

### PR DESCRIPTION
**Pull request by ralexstokes**
### What was wrong?

Lack of a code style tool.

### How was it fixed?

Introduces `black` and `isort` as a code formatter to the `eth2` code in this repo.

#### Cute Animal Picture

put a cute animal picture link inside the parentheses

### CODE REVIEW

File: eth2/_utils/bitfield.py [hunk 1]
```diff
@@ -1,6 +1,4 @@
-from cytoolz import (
-    curry,
-)
+from cytoolz import curry
```
Comment (hwwhww) [inline eth2/_utils/bitfield.py]: Per https://github.com/ethereum/snake-charmers-tactical-manual/blob/master/style-guide.md#import-statements , any chance to allow it?

Comment (ralexstokes) [reply eth2/_utils/bitfield.py]: which part exactly? i think we do want to add in use of `isort` but the default configuration b/t `isort` and `black` conflicted... in general, the `black` philosophy is that you can't configure it

Comment (hwwhww) [reply eth2/_utils/bitfield.py]: Sorry, the accurate link is here, formatting part: https://github.com/ethereum/snake-charmers-tactical-manual/blob/master/style-guide.md#formatting

> Multiple imports must be split across multiple lines, one per line.

Comment (pipermerriam) [reply eth2/_utils/bitfield.py]: Yeah, that's one that should be relaxed/removed.  Too strict and better to just let isort/black deal with it.

Comment (ralexstokes) [reply eth2/_utils/bitfield.py]: i'm in favor of "let isort/black deal with it"

**Comment by ralexstokes** [comment]
i can go back and add `isort` in this PR as the combo is what we want and i had left it out of this PR because I was confused about how they interacted.

**Comment by ralexstokes** [comment]
Closed in favor of #974.

**Pull request closed without merging**